internal/cli/users.go

```diff
@@ -0,0 +1,114 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+func usersCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "users",
+		Short: "Manage resources for users",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(userBlocksCmd(cli))
+	return cmd
+}
+
+func userBlocksCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "blocks",
+		Short: "Manage brute-force protection user blocks.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUserBlocksCmd(cli))
+	cmd.AddCommand(deleteUserBlocksCmd(cli))
+	return cmd
+}
+
+func listUserBlocksCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		userID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List brute-force protection blocks for a given user",
+		Long: `List brute-force protection blocks for a given user:
+
+auth0 users blocks list <User ID>
+`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				inputs.userID = args[0]
```

Comment:
Can you please ask the user to input the User Id? Like:

```go
if len(args) == 0 {
    if err := userID.Ask(cmd, &inputs.userID); err != nil {
	return err
    }
} else {
    inputs.userID = args[0]
}
```

You'd have to declare the userID input outside the command (eg at the top):

```go
appID = Argument{
    Name:       "User ID",
    Help:       "Id of the user.",
    IsRequired: true,
}
```

Reply:
<img width="465" alt="Screen Shot 2021-04-23 at 2 22 19 PM" src="https://user-images.githubusercontent.com/59627793/115931157-81a68180-a43f-11eb-98fb-81895e84f4fa.png">